docs/Project.toml

```
@@ -4,5 +4,5 @@ MathematicalSystems = "d14a8603-c872-5ed3-9ece-53e0e82e39da"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-Documenter = "≥ 0.23.0"
+Documenter = "0.23.0"
```

```suggestion
Documenter = "^ 0.23.0"
```
I thought you prefer this syntax.